jvm/ml4ir-inference/src/main/scala/ml4ir/inference/tensorflow/utils/ProtobufUtils.scala

```diff
@@ -30,73 +43,62 @@ case class SequenceExampleBuilder(config: FeatureConfig = FeatureConfig()) {
 
   /**
     * Functional API allowing the builder to act like a function to transform query/documents into a scorable protobuf
-    * @param query struct primarily containing the query text
+    * @param context struct primarily containing the query text
     * @param docs array of document-feature structs
     * @return TensorFlow's protobuf structure containing the raw features in one SequenceExample packet
     */
-  def apply(query: QueryContext, docs: Array[Document]): SequenceExample = {
+  def apply(context: Example, docs: Array[Example]): SequenceExample = {
```

Comment:
'examples' maybe? or will it be too abstract?

Reply:
I like it, that's better.